core/controllers/profile.py

```diff
@@ -76,16 +78,37 @@ def get(self, username):
         user_settings = user_services.get_user_settings_from_username(username)
         if not user_settings:
             raise self.PageNotFoundException
+        
+        user_contributions = user_services.get_user_contributions(
+            user_settings.user_id)
 
         self.values.update({
             'user_bio': user_settings.user_bio,
             'first_contribution_datetime': (
-                utils.get_time_in_millisecs(user_settings.first_contribution_datetime)
+                utils.get_time_in_millisecs(
+                    user_settings.first_contribution_datetime)
                 if user_settings.first_contribution_datetime else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(
+                user_contributions.created_explorations),
+            'edited_explorations_count': len(
+                user_contributions.edited_explorations)
+
         })
         self.render_json(self.values)
 
+    def get_public_contributions(list_explorations):
+        """Pass in created or edited explorations,
+        returns collection with only public explorations in list. 
+        """
+        public_explorations = []
+
+        for exploration_id in list_explorations:
```

Comment:
s/list_explorations/exploration_ids

Also - what happens if the user has a deleted exploration? With this implementation, I suspect you'll run into a server error, since the rights_manager.is_exploration_public() call will not be able to locate some of the explorations. Instead, I suggest using exp_services.get_exploration_summaries_matching_ids(), since you're probably going to be passing the exploration summaries to the frontend anyway.

Finally, this method is quite an important one, and needs tests.